### ISSUE TITLE
colors & gradient numerical values

### ISSUE THREAD

**Issue by nb-mao** [iskandarov-egor/mandelset#1]
Hi
Mandelset is really good and the only thing I miss is the possibility to see numerical values of the colors/gradient or save/import gradients.
Do you plan to add labels showing numerical values for gradient colours ans stops? It would be very useful to have the gradient in numerical form in case one wants to reuse it another time. 
Attached a few scaled down pics of my experiments.
Thanks. N
![mandelset_07](https://user-images.githubusercontent.com/89225599/130149683-d2d4267c-15e7-4ff1-8eff-910bff04f75d.png)
![mandelset_02](https://user-images.githubusercontent.com/89225599/130149663-56c6934d-ce1c-47dc-9c41-cdcc45dc8ed9.png)
![mandelset03](https://user-images.githubusercontent.com/89225599/130149635-29921de7-3e53-41e4-ad84-9c030b31d957.png)
![mandelset](https://user-images.githubusercontent.com/89225599/130149609-78bf53f2-c714-4d29-afee-13a6c70f704f.png)

**Comment by iskandarov-egor** [comment]
Hi, very nice pictures! it's good to know that someone uses the app. Can you tell me how you found it? I agree that the ability to save settings and gradients is a feature that the app is really missing. I will implement it in the future when I have more time. Thanks for the feedback!

**Comment by nb-mao** [comment]
Hi Thanks for your reply. I found it on searching for mandelbrot/fractals directly on github . I researched a lot and I found this app one of the best, it does exactly what I need. To get a nice image the most important is to get the right gradient and the lack of possibility to save it or at least see or input the numerical values is the only feature I miss. It is very very fast, light, responsive and easy to use compared to many others. I really like the feature to change colouring without re-rendering. I'm using it on an iMac with Apache on Fedora and Firefox browser. Max resolution I saved an image is 9000x4000px. Trying 11000x5000 now at 2e-33 the max where pan still works. Zoom limit is around e-36 e-37 when you start getting a pixelated cross artefact. Maybe more colouring options but beside that a really good app. I'm really looking forward to seeing new features!
